python/mxnet/module/bucketing_module.py

```diff
@@ -161,7 +161,7 @@ def output_shapes(self):
         assert self.binded
         return self._curr_module.output_shapes
 
-    def get_params(self):
+    def get_params(self, copy_to_cpu=True):
```

Comment:
Why is this copy_to_cpu argument needed here since it is not used. Nor is it declared the BaseModule

Reply:
It's there, because it's needed to be called by the user here during MXNet training: https://github.com/ctcyang/horovod/blob/0deb0d5d5c6befda6bad49ff713a0e0061fc08da/examples/mxnet_imagenet_resnet50.py#L266

Horovod only supports (input on GPU, output on GPU) type of communication. However, we found that `get_params` returns it as an NDArray on the CPU by default. That is why we need to add this argument here to make Horovod work. Since the default is set to `True`, it maintains backward compatibility with the rest of MXNet code.

Reply:
Please check Line 174 in this file: params = self._curr_module.get_params(). Do we need to change to params = self._curr_module.get_params(copy_to_cpu)?